src/components/tekstomrade/tekstomrade.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import 'nav-frontend-skjema-style';
+import classNames from 'classnames';
+import './tekstomrade.less';
+import { HighlightRule, LinkRule, BoldRule, ParagraphRule } from './parser/rules';
+import { parse, build } from './parser/parser';
+import { Rule } from './parser/domain';
+
+export * from './parser/rules';
```

Comment:
har ikke sett denne notasjonen før, hva skjer her..?

Reply:
Dette er re-eksportering av ting fra `./parser/rules`.
`./parser/rules` eksporterer alle reglene som er laget til til det nye tekstområdet, men for å slippe å vite om dette når man tar ibruk tekstområde komponenten så re-eksporteres de også fra `tekstomrade.tsx`.

På den måten kan man senere skrive `import Tekstomrade, { LinkRule } from './tekstomrade`
istedetfor `import Tekstomrade from './tekstomrade` og `import { LinkRule } from './parser/rules'`

Reply:
Wooow, avanserte greier ;-)